tbx/project_styleguide/templates/patterns/styleguide/typography/typography.html

```diff
@@ -0,0 +1,54 @@
+<div class="p-4 mb-8" style="max-width: 710px">
+    {# Headings #}
+    <p class="underline mt-8">Heading 1</p>
+    <h1 class="heading heading--one">Joining the battle to protect our oceans</h1>
+    <p class="underline mt-8">Heading 1-B</p>
+    <h1 class="heading heading--one-b">Joining the battle to protect our oceans</h1>
+    <p class="underline mt-8">Heading 2</p>
+    <h2 class="heading heading--two">Joining the battle to protect our oceans</h2>
+    <p class="underline mt-8">Heading 3</p>
+    <h3 class="heading heading--three">Joining the battle to protect our oceans</h3>
+    <p class="underline mt-8">Heading 4</p>
+    <h4 class="heading heading--four">Joining the battle to protect our oceans</h4>
+    <p class="underline mt-8">Heading 5</p>
+    <h5 class="heading heading--five">Joining the battle to protect our oceans</h5>
+    <p class="underline mt-8">Heading 6</p>
+    <h6 class="heading heading--six">Joining the battle to protect our oceans</h6>
+
+    {# Rich text #}
+    <p class="mt-8 underline">Rich text</p>
+    <div class="rich-text">
+        <h3>Heading 3</h3>
+        <h4>Heading 4</h4>
```

Comment:
Praise: great idea to add this styleguide.

I think we will need a style for heading 2s in the rich text block, as I have checked and we do have h2s in the rich-text block in the existing build.